packages/cli/internal/pkg/environment/environment.go

```diff
@@ -9,6 +9,11 @@ import (
 const DefaultEcrRegistry = "555741984805"
 const DefaultEcrRegion = "us-east-1"
 
+// TODO: Implement better tag versioning system
+const DefaultCromwellTag = "2021-10-01T21-33-26Z"
+const DefaultNextflowTag = "2021-10-01T21-33-26Z"
+const DefaultWesTag = "2021-10-01T21-33-26Z"
```

Comment:
Will these need to be updated periodically?

Reply:
Not really. With our new CI/CD process each engine has its own build pipeline. We'll only need to update them when we want to upgrade the engine version. Once the pipelines are set up we'll point AGC at that account/repository and specify a version number (e.g. "64" for Cromwell)